pkg/reflector/reflector_test.go

```diff
@@ -273,6 +273,7 @@ func TestEmitMetricFromFile(t *testing.T) {
 			nil,
 		},
 	} {
+		test := tt
```

Comment:
Ah, this is because there are implicit go routines being run with `r.Run()`, and the `t.Parallel()` tells the testing framework to run these ones all in parallel, whereas by default they would be executed sequentially.

Your fix here definitely smelled like the general "go routine inside of loop" problem, so I was curious exactly how it applied here.

https://gist.github.com/posener/92a55c4cd441fc5e5e85f27bca008721

Reply:
yea, exactly.

Reply:
yea, saw this mentioned in 100 mistakes in go book too, specifically mistake `#85`